molecule_common/base/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM centos/systemd:latest
```

Comment:
Agreed, if we can find another (more stable) tag to use, I'll be happy to switch to it.